models/chainlink/arbitrum/chainlink_arbitrum_ocr_reverted_transactions.sql

```diff
@@ -0,0 +1,62 @@
+{{
+  config(
+    tags=['dunesql'],
+    alias=alias('ocr_reverted_transactions'),
+    partition_by=['date_month'],
+    materialized='incremental',
+    file_format='delta',
+    incremental_strategy='merge',
+    unique_key=['hash', 'index', 'from'],
+    post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "chainlink",
+                                \'["linkpool_ryan"]\') }}'
+  )
+}}
+
+{% set incremental_interval = '7' %}
+
+WITH
+  arbitrum_usd AS (
+    SELECT
+      minute as block_time,
+      price as usd_amount
+    FROM
+      {{ source('prices', 'usd') }} price
+    WHERE
+      symbol = 'ETH'
+      {% if is_incremental() %}
+        AND minute >= date_trunc('day', now() - interval '{{incremental_interval}}' day)
+      {% endif %}      
+  ),
+  ocr_reverted_transactions AS (
+    SELECT
+      MAX(tx.block_time) as block_time,
+      date_trunc('month', MAX(tx.block_time)) as date_month,
```

Comment:
`date_trunc` function isn't enough to cast data types. you must explicitly cast to date. since you use `block_time`, it carries over timestamp type and our partitions won't allow that.